content/en/community/community-meeting.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Monthly Community Meeting
```

Comment:
This doesn't need to be committed. It'll be synced automatically when the site is generated 👍